src/components/Main.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
-import img1 from '../img/Manos_grande.png';
+import img1 from '../img/Illegal.png';
```

Comment:
It was requested from the group to put the image with the hands. I think we should ask before changing it.

Reply:
ahh I didn't know that, maybe yeah it's a good idea ask them but maybe we can change it and put it in materials for now and merge still?

Reply:
we can send them the link and they can decide.

Reply:
Can you please do that? Maybe just a screenshot from both? Thx